TesseractOCR/G8Tesseract.mm

```diff
@@ -644,6 +645,49 @@ - (NSString *)recognizedHOCRForPageNumber:(int)pageNumber {
     return nil;
 }
 
+- (NSData *)recognizedPDFForImages:(NSArray*)images {
```

Comment:
Ah yes, I agree that being able to render copyable text is a very good reason to use Tesseract's PDF renderer, thanks for letting me know that. :)